packages/block-editor/src/store/actions.js

```diff
@@ -1182,8 +1182,11 @@ export const mergeBlocks =
  * the set of specified client IDs are to be removed.
  *
  * @param {string|string[]} clientIds      Client IDs of blocks to remove.
- * @param {boolean}         selectPrevious True if the previous block should be
- *                                         selected when a block is removed.
+ * @param {boolean}         selectPrevious True if the previous block
```

Comment:
I know it's from before but can we change to `[selectPrevious=true]`  and reword this a bit.. 
Maybe something like: 
`Whether to select the previous block or the immediate parent if no previous block exists when a block is removed`.

Reply:
Applied Dave's suggestion